lib/routers/profile/pil.js

```diff
@@ -98,6 +98,11 @@ router.param('pilId', (req, res, next, id) => {
   Promise.resolve()
     .then(() => {
       return PIL[queryType]().findById(id)
+        .select([
+          'pils.*',
+          'profile.pilLicenceNumber as licenceNumber'
+        ])
```

Comment:
We probably need to handle this _a little_ more cleverly for the cases where the user has multiple old PIL records that do not have the same licence number.

i.e. load the profile licence number and the PIL licence number independently, and then if they differ respond with the one from the PIL.

Reply:
good shout, I hadn't thought of that